SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,34 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember([FromBody] User user)
+        {
+            if (user.Username == null)
+            {
+                return new BadRequestObjectResult("No empty fields allowed");
```

Comment:
Why do you return a `BadRequestObjectResult` instead of `BadRequest`?

Reply:
i thought that they are not different?